app/views/staff/shared/transactions/_table.html.haml

```diff
@@ -33,4 +33,4 @@
           %td.govuk-table__cell= transaction.receiving_organisation_name
           %td.govuk-table__cell
             - if policy(transaction).create?
-              = a11y_action_link(I18n.t('generic.link.edit'), edit_activity_transaction_path(transaction.parent_activity_id, transaction))
+              = a11y_action_link(I18n.t('generic.link.edit'), edit_activity_transaction_path(transaction.parent_activity_id, transaction), t("page_content.transactions.edit_noun", reference: transaction.reference))
```

Comment:
I am just about to remove Transaction references :cry: https://trello.com/c/Azapqja1/554-remove-reference-from-create-a-transaction